src/app/shared/module/poe/provider/stats-indistinguishable.provider.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core'
+import {
+  crafted,
+  delve,
+  enchant,
+  explicit,
+  fractured,
+  implicit,
+  monster,
+  pseudo,
+  veiled,
+} from '../../../../../assets/poe/stats-indistinguishable.json'
+import { StatIndistinguishableMap, StatType } from '../type'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StatsIndistinguishableProvider {
+  public provide(group: StatType): StatIndistinguishableMap {
```

Comment:
I think it might be useful to rename this to something like `getIndistinguishableStats` or similar.  Open for other suggestions though.

Reply:
This class is just a renamed copy of the same class that's used for the stats-local. For consistency sake I'd argue to refactor both, or neither. If a refactor would be favoured, I'd rather do that in a separate pull request as it has not much to do with the scope of this one.